standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/parser/Filter.g

```diff
@@ -70,6 +84,19 @@ import java.util.regex.Pattern;
     }
   }
 
+  public static java.sql.Timestamp ExtractTimestamp (String input) {
```

Comment:
Please start the method name with lowercase letter. `ExtractDate` is also wrong.

Reply:
ack